api/v1/latency.go

```diff
@@ -0,0 +1,71 @@
+package api
+
+import (
+	"encoding/json"
+	"net/http"
+	"time"
+
+	"github.com/celestiaorg/bittwister/xdp/latency"
+	"go.uber.org/zap"
+)
+
+// LatencyStart implements POST /latency/start
+func (a *RESTApiV1) LatencyStart(resp http.ResponseWriter, req *http.Request) {
+	var body LatencyStartRequest
+	if err := json.NewDecoder(req.Body).Decode(&body); err != nil {
+		sendJSONError(resp,
+			MetaMessage{
+				Type:    APIMetaMessageTypeError,
+				Slug:    SlugJSONDecodeFailed,
+				Title:   "JSON decode failed",
+				Message: err.Error(),
+			},
+			http.StatusBadRequest)
+		return
+	}
+
+	if a.lt == nil {
+		a.lt = &netRestrictService{
+			service: &latency.Latency{
+				Latency: time.Duration(body.Latency) * time.Millisecond,
+				Jitter:  time.Duration(body.Jitter) * time.Millisecond,
+			},
+			logger: a.logger,
+		}
+	} else {
+		lt, ok := a.lt.service.(*latency.Latency)
```

Comment:
same comment here as above, removing the else condition adds a built in type check to the code in the if condition.

Reply:
responded [here](https://github.com/celestiaorg/bittwister/pull/23/files#r1413781466)